src/series/bar.js

```diff
@@ -86,25 +145,32 @@ export default function() {
         yScale = x;
         return bar;
     };
-    bar.xValue = function(x) {
+    bar.y0Value = function(x) {
         if (!arguments.length) {
-            return xValue;
+            return y0Value;
         }
-        xValue = x;
+        y0Value = d3.functor(x);
         return bar;
     };
-    bar.y0Value = function(x) {
+    bar.x0Value = function(x) {
         if (!arguments.length) {
-            return y0Value;
+            return x0Value;
         }
-        y0Value = d3.functor(x);
+        x0Value = x;
```

Comment:
> Any reason y0Value is a functor and x0value/yValue/xValue aren't?

Because my code snippet for properties doesn't add functors ;-)

Good spot. I've made `x0Value` a functor property in line with the current bar implementation. I can 't see any reason for using a functor for `xValue` or `yValue`.